services/openshiftmisc/src/handlers/resticRestore.ts

```diff
@@ -75,11 +75,19 @@ export async function resticRestore (data) {
   });
 
   try {
+    let baasBucketName = ""
+    for (let variable of result.envVariables) {
```

Comment:
The result of `getOpenShiftInfoForProject` is an object keyed by `project` so you'd want to do `result.project.envVariables` or use `projectOpenShift.envVariables`.